containers/init/build/php7/Dockerfile

```diff
@@ -21,12 +21,12 @@ RUN apt-get update && apt-get install -y \
   zlib1g-dev \
   build-essential \
   lcov \
-  make && \
+  make \
+  gnugp2 && \
   apt-get clean
 
 # Install rvm
-RUN apt-get update && apt-get install -y gnupg2 && apt-get clean
-RUN gpg2 --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+RUN gpg2 --recv-keys 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
This modification would make gpg2 ask embedded key server for public key?

Reply:
Yes this is the same as what happens for interop.

https://github.com/grpc/grpc/blob/73dcfd375d14c7cef537a5255767a8950424988c/tools/dockerfile/interoptest/grpc_interop_ruby/Dockerfile#L73